src/utils/address.js

```diff
@@ -0,0 +1,30 @@
+import { validateName } from './utils'
+
+export const ETH_ADDRESS_TYPE = {
+  name: 'name',
+  address: 'address',
+  error: 'error'
+}
+
+export function getEthAddressType(address) {
```

Comment:
Can we not just use this? 
https://web3js.readthedocs.io/en/1.0/web3-utils.html#isaddress